### PR TITLE
Added UID and bootstrap url parameters

### PR DESCRIPTION
**Pull request by ddd-mtl**

### CODE REVIEW

**Comment by Connoropolous** [comment]
@ddd-mtl are you using this binary??

File: src/embedded_runner.rs [hunk 1]
```diff
@@ -197,7 +206,7 @@ async fn install_or_passthrough(
     }
 
     emit(&event_channel, StateSignal::IsReady).await;
-    println!("APP_WS_PORT: {}", using_app_ws_port);
+    println!("     APP_WS_PORT: {}", using_app_ws_port);
```
Comment (Connoropolous) [inline src/embedded_runner.rs:209]: ? just wondering

File: src/main.rs [hunk 2]
```diff
@@ -58,7 +65,7 @@ fn main() {
         // we use both IO and Time tokio utilities
         .enable_all()
         // give our threads a descriptive name (they'll be numbered too)
-        .thread_name("acorn-tokio-thread")
+        .thread_name("hc-runner-tokio-thread")
```
Comment (Connoropolous) [inline src/main.rs]:
```suggestion
        .thread_name("holochain-runner-tokio-thread")
```

good point I think maybe we should go full title match tho

**Review by Connoropolous — CHANGES_REQUESTED** [review]
sweet, makes good sense. 

can you delete `test/data/.txt`

File: src/embedded_runner.rs (hunk 1 above)
Comment (ddd-mtl) [reply src/embedded_runner.rs:209]: Just aligning the output with the next println

**Review by Connoropolous — APPROVED** [review]
Great

**Comment by Connoropolous** [comment]
I can publish today

**Pull request merged**